crates/hdk/src/host_fn/call.rs

```diff
@@ -1 +1,26 @@
-//! @todo
+use crate::prelude::*;
+use holo_hash::AgentPubKey;
+use holochain_wasmer_guest::SerializedBytesError;
+use holochain_zome_types::{
+    call::Call,
+    zome::{FunctionName, ZomeName},
+};
+
+use crate::host_fn;
+
+pub fn call(
```

Comment:
I notice this is a function rather than a macro. Don't get me wrong -- the fewer exclamation points the better, esp for people who are using RLS for code intel -- but in order to be consistent with the other host functions, should it stay a macro for now?

Reply:
We are planning to make them all function calls very soon

Reply:
I'd rather just change them all to functions then add in macros to then change back. It's not a big job to change them

Reply:
That'd be lovely. Another thing that I've discovered in my "audit of RSM documentation" is that macros thwart rustdoc's ability to do auto-linking of argument and return types, which is painful for dev ergonomics.